core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -555,6 +557,30 @@ public final void setTlsEnabled(final boolean enabled) {
         this.tlsEnabled = enabled;
     }
 
+    /**
+     * Get the host override for TLS SNI Server Name set by {@link #setSniServerName(String)}.
+     *
+     * @return  the sniServerName
```

Comment:
Please start the description with an upper case letter and finish with a period (`.`)